### PR TITLE
Add docs for AllRelationshipsScan

### PR DESCRIPTION
**Pull request by pontusmelke**

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/execution-plan-groups/operators.asciidoc [hunk 1]
```diff
@@ -48,6 +48,11 @@ include::../ql/query-plan/undirected-relationship-index-seek-by-range.asciidoc[]
 // UnionNodeByLabelsScan
 include::../ql/query-plan/union-node-by-labels-scan.asciidoc[]
 
+/ DirectedAllRelationshipsScan
```
Comment (pontusmelke) [inline cypher/cypher-docs/src/docs/dev/execution-plan-groups/operators.asciidoc:51]: I stole this from https://github.com/neo4j/neo4j-documentation/pull/1507, it used to be the case that this was not a necessary step but happened automatically somehow but I guess not you have to do it manually?

File: cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc [hunk 2]
```diff
@@ -129,6 +129,12 @@ Tests for the absence of a pattern predicate.
 | label:yes[]
 |
 
+| <<query-plan-directed-all-relationships-scan,DirectedAllRelationshipsScan>>
```
Comment (pontusmelke) [inline cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc:132]: I stole this from https://github.com/neo4j/neo4j-documentation/pull/1507, it used to be the case that this was not a necessary step but happened automatically somehow but I guess not you have to do it manually?

File: cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc [hunk 3]
```diff
@@ -609,6 +615,12 @@ Tests for the absence of a pattern predicate if an expression predicate evaluate
 |
 |
 
+| <<query-plan-undirected-all-relationships-scan,UndirectedAllRelationshipsScan>>
```
Comment (pontusmelke) [inline cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc:618]: I stole this from https://github.com/neo4j/neo4j-documentation/pull/1507, it used to be the case that this was not a necessary step but happened automatically somehow but I guess not you have to do it manually?

**Comment by pontusmelke** [comment]
[Document AllRelationshipsScan](https://trello.com/c/Cep9QaUR/3694-document-allrelationshipsscan)

**Review by AlexicaWright — APPROVED** [review]
LGTM 👍

**Pull request merged**